lib/gui/app/components/settings/.eslintrc.yml

```diff
@@ -0,0 +1,2 @@
+parserOptions:
+  sourceType: module
```

Comment:
To start using module syntax (e.g. `import`) in `js | jsx` files too. Feels nicer to me as we can move small bits to modules, then switch completely in the main compiler too when we're done

Reply:
Or we can just refactor said files into typescript, of course. I started putting this to avoid the need to rewrite every file to typescript as it's more than just `use strict` and `export | import`